torch_lr_finder/lr_finder.py

```diff
@@ -433,7 +433,7 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None, su
                 shown . Default: None.
             suggest_lr (bool, optional): suggest a learning rate by
                 - 'steepest': the point with steepest gradient (minimal gradient)
-                you can use that point as a first guess for an LR. Default: False.
+                you can use that point as a first guess for an LR. Default: True.
 
         Returns:
```

Comment:
Update the `Returns` docstring describing that the suggested learning rate is returned if `suggest_lr=True`

Reply:
Sorry, fixing...